README.md

```diff
@@ -53,23 +55,33 @@ and processor architecture listed below.
 * CentOS 8
 * Debian 9
 * Debian 10
-* FreeBSD 11.3
+* FreeBSD 11.4
 * FreeBSD 12.1
-* Ubuntu 16.04
-* Ubuntu 18.04
+* Ubuntu 16.04.7
+* Ubuntu 18.04.5
+* Ubuntu 20.04.1
 
 ### Supported database engine versions
 
-Operating System | MySQL/MariaDB   | PostgreSQL
+Operating System | MariaDB         | PostgreSQL
 ---------------- | ----------------| -----------
-CentOS 7         | 5.6 (MySQL)     | 9.3
-CentOS 8         | 8.0 (MySQL)     | 10.6
-Debian 9         | 10.1 (MariaDB)  | 9.6
-Debian 10        | 10.3 (MariaDB)  | 11.7
-FreeBSD 11.3     | 5.7 (MySQL)     | 11.7
-FreeBSD 12.1     | 5.7 (MySQL)     | 11.7
-Ubuntu 16.04     | 10.0 (MariaDB)  | 9.5
-Ubuntu 18.04     | 10.1 (MariaDB)  | 10.12
+CentOS 7         | 5.5 (#)         | 9.3 (##)
+CentOS 8         | ????            | 10.6 (##)
+Debian 9         | 10.1            | 9.6
+Debian 10        | 10.3            | 11.7
+FreeBSD 11.4     | 10.5.5 (###)    | 12.4
+FreeBSD 12.1     | 10.5.5 (###)    | 12.4
+Ubuntu 16.04.7   | 10.0            | 9.5
+Ubuntu 18.04.5   | 10.1            | 10.12
+Ubuntu 20.04.1   | 10.3 (#)        | 12.4 (#)
```

Comment:
The third decimal for Ubuntu is not needed, I think.

Reply:
I think functionally the third level on Ubuntu is more like the second level of FreeBSD. You cannot refer to version Ubuntu 18, because 18.04 and 18.10 are two different versions, like FreeBSD 11 and FreeBSD 12.

Maybe we should just say FreeBSD 11, but there could be more than subversion of 11 at the same time. Currently we have both 11.3 and 11.4 as supported versions (even though 11.3 is soon deprecated).

I will remove the subversion of Ubuntu. We do not specify it for neither Debian nor CentOS.

Reply:
I don't think it helps to be too granular about what releases we support. No one will dig through archives anyway just to download the exact point-release we specified in our documentation. And if you update your OS, as you should, you still get something slightly different anyway.

Reply:
I have updated the file and only indicate one level on Linux OSs. (I consider 18.04 for Ubuntu to  one level, comparable with 9 for Debian.) For FreeBSD I have kept the second level, as we did last time. Is that fine?